manifests/client.pp

```diff
@@ -36,6 +36,7 @@
   Optional[Stdlib::Absolutepath] $tls_cacertdir = undef,
   $tls_checkpeer                                = undef,
   $tls_reqcert                                  = undef,
+  Variant[Openldap::Tls_moznss_compatibility, 'absent', Undef] $tls_moznss_compatibility = undef,
```

Comment:
It doesn't make it much shorter indeed… 🤷 Nevermind, let's keep it this way, I guess 😁

Reply:
Unfortunately, this `Variant` doesn't seem to work. See [the tests](https://travis-ci.org/github/camptocamp/puppet-openldap/jobs/755724000#L39663):

```
Error creating type specialization of a Variant-Type, Cannot use String where Any-Type is expected
```

🤔

Reply:
Ah yes, I get it: `Variant` takes types, not strings (which would require an `Enum`). So you might need to include `'absent'` in the type itself (which arguably is not very elegant, but would also allow to use `Optional[Openldap::Tls_moznss_compatibility]` instead).

Reply:
Argh. I can't tell when it's "Travis actually failed" or "Travis timed out" anymore, or I would've fixed this over the weekend.
Yeah, I don't liiike that, but I'll do it.  The only other option I can think of is a multi-liner in `client.pp`.

Reply:
Yeah, I need to get away from Travis. I have "migrate Puppet modules to GitHub Actions" in my TODO list.